.circleci/config.yml

```diff
@@ -91,7 +91,7 @@ jobs:
           command: |
             echo -e "[pypi]" >> ~/.pypirc
             echo -e "username = solegalli" >> ~/.pypirc
-            echo -e "password = $PYPI_PASSWORD" >> ~/.pypirc
+            echo -e "password = $pypi_password" >> ~/.pypirc
```

Comment:
should work, but more standard is with curly braces: `${pypi_password}` caps is also preferred for evars (https://circleci.com/docs/2.0/env-vars/)